app/models/concerns/smooch_messages.rb

```diff
@@ -443,7 +443,9 @@ def smooch_relate_items_for_same_message(message, associated, app_id, author, re
         if text_words > self.min_number_of_words_for_tipline_long_text
           # Remove link from text
           link = self.extract_url(message['text'])
-          message['text'] = message['text'].remove(link.url)
+          if link  && link.respond_to?(:url)
```

Comment:
Minor: Two spaces between `link` and `&&` - please fix to one.

Reply:
Thank you, fixed!